bitnami/minio/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: minio
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/minio
-version: 12.10.10
+version: 12.10.11
```

Please note you're adding a new parameter. Therefore, you need to bump the minor version:
```suggestion
version: 12.11.0
```